app/assets/stylesheets/styles/components/cards/_cards-protection-dials.scss

```diff
@@ -0,0 +1,46 @@
+&--protection-dials {
+  @include flex;
+  @include flex-h-between;
+  margin-top: rem-calc(80);
+  margin-bottom: rem-calc(50);
+
+  .card {
+    padding: rem-calc(0 10);
+    width: 20%;
+
+    flex: 1 1 20%;
+
+    &--unknown {
+      .card__habitat-status {
+        opacity: .4;
+      }
+    }
+
+      &__dial {
+        margin-bottom: rem-calc(10);
+      }
+
+      &__habitat {
+        text-align: center;
+      }
+
+        &__habitat-title {
+          @include text-protection-dials-title;
+          margin-bottom: rem-calc(22);
+        }
+
+        &__habitat-status {
+          @include text-protection-dials-status;
+
+          display: block;
+        }
+
+          &__habitat-percentage {
+            color: $grey;
+            font-size: rem-calc(22);
+
+            display: block;
+          }
+
```

Comment:
I think the indentation here is still off.
I would put all rules at the same indentaion level if they are not nested